spec/requests/api/v3/courses/index_spec.rb

```diff
@@ -227,6 +227,25 @@
           expect(course_hashes.second["id"]).to eq(course_a.id.to_s)
         end
       end
+
+      context "when a course has a site that has not been geocoded" do
```

Comment:
Can we push this test further down? I think a test that tests the service + db would be sufficient and faster.

Reply:
I can do that next time. It would only be slightly faster as serialisation is not performed. I don't feel pushing this particular working test down at this moment in time is worthwhile.

Reply:
This :point_right: https://martinfowler.com/articles/practical-test-pyramid.html#TheTestPyramid

But I won't hold up the PR if you don't want to do it.